getting_started/Data_Input_Pipeline.md

```diff
@@ -96,7 +96,7 @@ bash download_dataset.sh {GCS_PROJECT} {GCS_BUCKET_NAME}
 ```
 2. Use the following config:
 ```
-dataset_type: c4
+dataset_type: tfds
```

Comment:
I think this means you need to sync to main?

Reply:
dropped the tfds related commit. Will figure our how to sync.